Maths/CollatzSequence.js

```diff
@@ -26,5 +26,5 @@ export function collatz (n) {
     steps.push(n)
   }
 
-  return { result: n, steps: steps }
+  return { result: n, steps }
```

Comment:
This change is not related to this pr, so undo this.

Reply:
resolved